core/modules/system/system.install

```diff
@@ -3802,6 +3802,16 @@ function system_update_1102() {
   }
 }
 
+/**
+ * Show a recommendation to the user to check the configuration of their views
+ * access control settings.
+ *
+ * @see https://github.com/backdrop/backdrop-issues/issues/5919
+ */
+function system_update_1103() {
+  return t("Previously, when changing access settings on a view display it could unexpectedly change the setting on other non-overridden displays of that view. This issue has now been fixed. If you have views where different displays have different access rules, you are recommended to check that the settings under 'Access > Access' are correct for these views.");
+}
+
```

Comment:
This update hook is non-conventional and alarming. Let's take this out and just fix the bug.

```suggestion
```